brigade/static/scss/atoms/_list.scss

```diff
@@ -0,0 +1,15 @@
+@import '../core/variables';
+
+//
+// List
+//
+
+.list--inline {
+  list-style-type: none;
+}
+
+.list--inline li {
```

Comment:
I notice you didn't really nest/indent any of these in their parent rules as I would have expected. Is that on your todo list for the future? I think it makes the SASS a lot more readable to have it indented and included in the parent class.

Reply:
Definitely on the todo list!